pynamodb/expressions/condition.py

```diff
@@ -25,28 +25,28 @@ def serialize(self, placeholder_names, expression_attribute_values):
 
     def __and__(self, other):
         if not isinstance(other, Condition):
-            raise TypeError("unsupported operand type(s) for &: '{0}' and '{1}'",
-                            self.__class__.__name__, other.__class__.__name__)
+            raise TypeError("unsupported operand type(s) for &: '{0}' and '{1}'"
```

Comment:
nit - I think `pyupgrade` skipped the `{0}`s during its rewrite due to the omitted `.format`